pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -224,6 +236,85 @@ func RunCreate(cmd *cobra.Command, args []string) {
 	}
 }
 
+// RunUpdate runs a CLI command
+func RunUpdate(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <noobaa-account-name> %s`, cmd.UsageString())
+	}
+	name := args[0]
+
+	newDefaultResource, _ := cmd.Flags().GetString("new_default_resource")
+
+	o := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaaaccount_cr_yaml)
+	noobaaAccount := o.(*nbv1.NooBaaAccount)
+	noobaaAccount.Name = name
+	noobaaAccount.Namespace = options.Namespace
+
+	if newDefaultResource == "" {
+		log.Fatalf(`❌ New Default Resource must be provided as flag to update`)
+	}
+
+	isResourceBackingStore := checkResourceBackingStore(newDefaultResource)
+	isResourceNamespaceStore := checkResourceNamespaceStore(newDefaultResource)
+
+	if isResourceBackingStore && isResourceNamespaceStore {
+		log.Fatalf(`❌  got BackingStore and NamespaceStore %q in namespace %q`,
+			newDefaultResource, options.Namespace)
+	} else if !isResourceBackingStore && !isResourceNamespaceStore {
+		log.Fatalf(`❌ Could not get BackingStore or NamespaceStore %q in namespace %q`,
+			newDefaultResource, options.Namespace)
+	}
+
+	if !util.KubeCheck(noobaaAccount) && (name != "admin@noobaa.io") {
```

Comment:
The account `admin@noobaa.io` doesn't have CRD, it is internal.
having this condition means that the `admin@noobaa.io` is not updatable. 
```suggestion
	if !util.KubeCheck(noobaaAccount)  {
```

Reply:
Okay.